tests/suite/scripting/import.typ

```diff
@@ -55,6 +55,34 @@
 #test(name, "Klaus")
 #test(othername, "Klaus")
 
+--- import-items-parenthesized ---
```

Comment:
Please add one or two negative tests where the right paren is missing (maybe one in a single line and one multiline).

Reply:
Sure will do after working ;)

Reply:
Done, please let me know if more are needed